providers/vic/operations/pod_creator.go

```diff
@@ -92,10 +92,6 @@ func NewPodCreator(client *client.PortLayer, imageStore proxy.ImageStore, isolat
 		return nil, PodCreatorIsolationProxyError
 	} else if podCache == nil {
 		return nil, PodCreatorPodCacheError
```

Comment:
I thought you were only changing the error message.  I think we should still do the check but we can change the message.